README.md

```diff
@@ -1,9 +1,9 @@
 # Nautobot Floor Plan
 
 <p align="center">
-  <img src="https://raw.githubusercontent.com/nautobot/nautobot-plugin-floor-plan/develop/docs/images/icon-nautobot-floor-plan.png" class="logo" height="200px">
+  <img src="https://raw.githubusercontent.com/nautobot/nautobot-app-floor-plan/develop/docs/images/icon-nautobot-floor-plan.png" class="logo" height="200px">
   <br>
-  <a href="https://github.com/nautobot/nautobot-plugin-floor-plan/actions"><img src="https://github.com/nautobot/nautobot-plugin-floor-plan/actions/workflows/ci.yml/badge.svg?branch=main"></a>
+  <a href="https://github.com/nautobot/nautobot-app-floor-plan/actions"><img src="https://github.com/nautobot/nautobot-app-floor-plan/actions/workflows/ci.yml/badge.svg?branch=main"></a>
   <a href="https://docs.nautobot.com/projects/floor-plan/en/latest"><img src="https://readthedocs.org/projects/nautobot-plugin-floor-plan/badge/"></a>
```

Comment:
What's involved in changing readthedocs URL to `nautobot-app-floor-plan`?

Reply:
I _think_ it would just be updating the name for each project in RTD, @cmsirbu ?

Reply:
That part is so invisible (due to the sub-project assignment to docs.nautobot.com), it might not be worth it. But yes, it can be changed in the project settings.